OrbitCore/Capture.cpp

```diff
@@ -423,7 +423,7 @@ bool Capture::IsCapturing() {
 //-----------------------------------------------------------------------------
 TcpEntity* Capture::GetMainTcpEntity() {
   return Capture::IsRemote() ? (TcpEntity*)GTcpClient.get()
```

Comment:
maybe remove this cast as well?

Reply:
I used an if instead. Somehow the ternary operator does not like two different return types.